contrib/local-environment/nginx.conf

```diff
@@ -1,6 +1,6 @@
 # Reverse proxy to oauth2-proxy
 server {
-  listen       80;
+  listen       8080;
```

Comment:
What's the motivation behind changing the nginx bind?

Reply:
This is mainly for my test environment because the system port is blocked by an other service. I marked this patches with `[DO NOT MERGE]` because they need to be removed before merge.